src/service/follower.proto

```diff
@@ -7,6 +7,13 @@ import "blockchain_txn.proto";
 import "gateway_staking_mode.proto";
 import "region.proto";
 
+enum follower_error { gateway_not_found = 0; }
+
+message follower_error_resp {
```

Comment:
Why is this here? What would bytes contain?

Reply:
See @andymck ’s previous comment; contextual but in this case would be used to send back the pubkey that was missing in case the caller was making multiple calls and needed to differentiate which gateway wasn’t found

Reply:
Yeah, I really dislike the "blob of contextual data" approach here vs errors with actual details in the message type itself.

Reply:
I can remove it but I’m disinclined to gold-plate this since it’s such a short-lived solution. Lemme play around briefly with a more structured not_found error than just an enum

Reply:
adjusted here to make the error type more distinct: [0896e48](https://github.com/helium/proto/pull/209/commits/0896e488d62c86a2978b8546b268d5aa4608388d); lemme know what you think